compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -9,10 +9,12 @@
 
 include "mlir/Pass/PassBase.td"
 
```

Comment:
> Reorder the passes in Passes.td to be alphabetical within each of the following
portions

Can you add a comment explaining the ordering here? Otherwise I could see myself or others reordering it to alphabetical later on.